lib/katello/tasks/upgrades/4.1/sync_noarch_content.rake

```diff
@@ -0,0 +1,13 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '4.1' do
+      desc "Synchronize arches field of content in candlepin which should be unset."
+      task :sync_noarch_content => ['environment'] do
+        roots = Katello::RootRepository.joins(:product).merge(Katello::Product.custom).where(arch: 'noarch')
+        roots.each do |root|
+          Katello::Resources::Candlepin::Content.update(root.library_instance.organization.label, id: root.content_id, arches: '')
```

Comment:
In my dev environment the data is a little bit messed up. I had a RootRepository but the candlepin Content was gone which made this raise a 404. If any users out there have some bad data this would break upgrades. Perhaps this update call should rescue from RestClient::NotFound and log the error instead of failing
```
RestClient::NotFound: Katello::Resources::Candlepin::Content: 404 Not Found {"displayMessage":"Content with ID \"1617221979927\" could not be found.","requestUuid":"d191baf3-222b-47db-99a0-b7b959d46d55"} (PUT /candlepin/owners/import-5905/content/1617221979927)
```